website/docs/docs/dbt-versions/experimental-features.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Preview new and experimental features in dbt Cloud"
+id: "experimental-features"
+sidebar_label: "Preview new dbt Cloud features"
+description: "Gain early access to many new dbt Labs experimental features by enabling this in your profile."
+---
+
+dbt Labs often beta tests features before deciding to continue on the [Product lifecycle](https://docs.getdbt.com/docs/dbt-versions/product-lifecycles#dbt-cloud).
+
+You can access experimental features if you want to preview beta features that haven’t yet been released to dbt Cloud. You can toggle on or off all experimental beta features in one place. As with other beta features, these features:
```

Comment:
I think we have to be careful about using both `experimental` and `beta`
Experimental elicits the feeling that the feature is something we're messing around with but may never see the light of day.
Beta generally means a feature we're committed to but it's in the early stages of the lifecycle. 
I'm not sure we should use them interchangeably in the doc, even if that's what it is, as it kind of sends mixed messages.

Reply:
Maybe `alpha` would be a better term than `beta` for this doc. What are your thoughts @runleonarun?